ext/Rakefile

```diff
@@ -1,40 +1,55 @@
 # frozen_string_literal: true
 
 require File.expand_path('../../lib/rdkafka/version', __FILE__)
-require "mini_portile2"
 require "fileutils"
 require "open-uri"
 
 task :default => :clean do
-  # Download and compile librdkafka
-  recipe = MiniPortile.new("librdkafka", Rdkafka::LIBRDKAFKA_VERSION)
+  if ENV.fetch('RDKAFKA_EXT_PATH', '').empty?
```

Comment:
Usage-wise, it looks like will be up to the invoker to set `RDKAFKA_EXT_PATH`, is that correct?  So instead of running what the doc suggests:

```
cd ext && bundle exec rake && cd ..
```

They should run something like this under Nix:

```
cd ext && RDKAFKA_EXT_PATH=/some/path/user/provides/ bundle exec rake && cd ..
```

Is that correct?

Reply:
@alexandriainfantino would you mind ^^

Reply:
Hey!

`RDKAFKA_EXT_PATH` is a part of our nix configuration, it will be up to the invoker to ensure their nix configuration is set up appropriately

Reply:
Could we add a few lines to the documentation indicating what needs to be done in a Nix configuration?  Or maybe just add a comment here so that if someone searches the codebase for "Nix" they might get a hit?

I'm not trying to make a mountain out of a molehill, but it feels like if we are going to add support for a particular environment, we should make it as easy as possible for users of that environment to know we support it and how best to use it.

Not trying to lead the witness, but maybe something like:

```
raise "RDKAFKA_EXT_PATH must be set in your environment when running under nix" unless ENV["IN_NIX_SHELL"]
```

Reply:
Absolutely! will add

Reply:
pushed up a small check with comment 👍

Reply:
@geoff2k you good? Wanted to include this in the upcoming release and also backport to karafka-rdkafka (I do my best to sync them and retire karafka-rdkafka)